### PR TITLE
feat: added compat flag to modify debian compatibility level

### PR DESCRIPTION
**Pull request by veimox**
added compat flag to modify debian compatibility level

### CODE REVIEW

File: stdeb/util.py [hunk 1]
```diff
@@ -1248,6 +1252,7 @@ def build_dsc(debinfo,
               debian_dir_only=False,
               sign_dsc=False,
               ignore_source_changes=False,
+              compat=9
```
Comment (dirk-thomas) [inline stdeb/util.py:1255]: Shouldn't the code calling this function be updated to pass the configured compat version? Otherwise this would be a hard coded value again.

**Review by dirk-thomas — CHANGES_REQUESTED** [review]
Thank you for working on this feature. I rebased your branch after merging #158 which changed the default debian compatibility level to 9. I added one question inline.

**Comment by veimox** [comment]
@dirk-thomas re-rebased without the unnecessary flag

File: stdeb/command/common.py [hunk 2]
```diff
@@ -35,6 +35,7 @@ def initialize_options (self):
         self.with_dh_systemd = False
         self.sign_results = False
         self.ignore_source_changes = False
+        self.compat = 7
```
Comment (dirk-thomas) [inline stdeb/command/common.py:38]: Shouldn't this be a `9` too? Since the default value is now repeated in more than one place please create a global constant for it.

File: stdeb/util.py [hunk 3]
```diff
@@ -1253,7 +1257,7 @@ def build_dsc(debinfo,
               remove_expanded_source_dir=0,
               debian_dir_only=False,
               sign_dsc=False,
-              ignore_source_changes=False,
+              ignore_source_changes=False
```
Comment (dirk-thomas) [inline stdeb/util.py:1260]: Nitpick: please revert this unrelated change.

**Review by dirk-thomas — CHANGES_REQUESTED** [review]

**Comment by veimox** [comment]
@dirk-thomas sorry for that! commit updated!

File: stdeb/command/common.py (hunk 2 above)
Comment (dirk-thomas) [reply stdeb/command/common.py]: > Since the default value is now repeated in more than one place please create a global constant for it.

This comment is still pending. Can you please update the patch to define `DEFAULT_COMPAT = 9` somewhere and use it instead of the hard coded default. And also mention the default value in the help text as well as README. Thanks.

Comment (dirk-thomas) [reply stdeb/command/common.py]: @veimox Friendly ping (since I would like to get a new minor release out later this week).

**Comment by veimox** [comment]
@dirk-thomas I introduced `DH_DEFAULT_VERS` and add it to the documentation both the cli and the `README.md`

**Review by dirk-thomas — APPROVED** [review]

**Comment by dirk-thomas** [comment]
Thank you for the enhancement and for iterating on this.
